Survey/Sources/Presentation/Modules/SurveyQuestion/Views/SurveyQuestionBodyView.swift

```diff
@@ -6,26 +6,82 @@
 //  Copyright © 2022 Nimble. All rights reserved.
 //
 
+import Combine
 import SwiftUI
 
 struct SurveyQuestionBodyView: View {
 
+    @ObservedObject var input: SurveyQuestionBodyViewModel.Input
+    @ObservedObject var output: SurveyQuestionBodyViewModel.Output
+
+    private let loadTrigger = PassthroughSubject<Void, Never>()
+
     var body: some View {
         VStack(alignment: .leading) {
-            // TODO: Remove dummy Text
-            Text("1/5")
+            // MARK: - Remove dummy
```

Comment:
I think the line break here is unnecessary and TODO is more accurate than MARK
```suggestion
            // TODO: Remove dummy
```

Reply:
I fixed at https://github.com/nkhanh44/ic-survey-ios/pull/78/commits/0aa8d9464b34c2d7406787934d41b70938be2ada